packages/file-ops/src/AwsS3Fs.js

```diff
@@ -218,6 +218,10 @@ export default class AwsS3Fs {
       Delimiter: '/'
     }).promise()
 
+    const hasPlaceholderFile = Contents.find(item => item.Key.endsWith('.placeholder'))
```

Comment:
https://github.com/ObsidianLabs/electron-components/blob/9fc651fe120af8c127234329f94f030c78807c31/packages/file-ops/src/AwsS3Fs.js#L72

Reply:
After deleting all files in the folder, the folder needs to contain at least one .placeholder empty file in order for the folder not to be lost after filtering, so if there is no .placeholder empty file in the folder, one will be created automatically